contracts/dependencies/contracts.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity 0.5.0;
```

Comment:
this should be under `test/dependencies/contracts` or better yet, just use `package.json` 
if this was for development, remove them before removing 'not ready' label